core/src/types/base-object.ts

```diff
@@ -0,0 +1,16 @@
+import {Field, Int, ObjectType} from "@nestjs/graphql";
+
+@ObjectType()
+export class BaseObject {
+    @Field(() => Int)
+    id: number;
+
+    @Field(() => Date)
+    createdAt: Date;
+
+    @Field(() => Date, {nullable: true})
+    updatedAt: Date;
+
+    @Field(() => Date, {nullable: true})
+    deletedAt: Date;
```

Comment:
Nullable field needs `?`

```suggestion
    updatedAt?: Date;
```